src/backend/InvenTree/plugin/api.py

```diff
@@ -397,7 +400,8 @@ def get(self, request):
                     })
 
         result = PluginSerializers.PluginRegistryStatusSerializer({
-            'registry_errors': error_list
+            'registry_errors': error_list,
+            'active_plugins': PluginConfig.objects.filter(active=True).count(),
```

Comment:
Comment: Nice add

Reply:
Definitely, this was my plan on the longer run to add more status info here about plugins. And that's also why I added no dedicated errors endpoint, because I thought we can later extend this.